README.md

```diff
@@ -41,9 +41,12 @@ brew install move2kube
 
 To bring up UI version:
 
-* Change directory to an empty directory using say, `mkdir -p workspace && cd workspace`
-* Run `docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest`
-* Access the UI in `http://localhost:8080/`.
+```
+docker run --rm -it -p 8080:8080 -v "${PWD}/data:/move2kube-api/data" -v /var/run/docker.sock:/var/run/docker.sock quay.io/konveyor/move2kube-ui:latest
+```
+Then go to http://localhost:8080 in a browser
+
+More detailed instructions, including how to build the image from source, can be found here: https://github.com/konveyor/move2kube-ui#starting-the-ui
```

Comment:
Make 'here' a link, instead of full url.

Reply:
It's good practise not to hide links. People like to see where they are being taken to.